libraries/core-react/src/components/Snackbar/SnackbarAction.tsx

```diff
@@ -4,9 +4,9 @@ import { snackbar as tokens } from './Snackbar.tokens'
 
 const StyledSnackbarAction = styled.div`
   display: inline-flex;
-  margin-left: ${tokens.spacings.actionSpace};
-  margin-top: -10px;
-  margin-bottom: -10px;
+  margin-left: ${tokens.entities.actions.spacings.left};
+  margin-top: -${tokens.entities.actions.spacings.top};
+  margin-bottom: -${tokens.entities.actions.spacings.bottom};
```

Comment:
You can use `spacingsTemplate` here?

Reply:
It was because of the negative values mentioned above but I'll try with aligning with flex 👍

Reply:
Seem like it has to be this way because we don't want `16px` padding on the buttons 🤔